presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -350,8 +351,12 @@ protected RelationType visitDescribeInput(DescribeInput node, AnalysisContext co
         // get all parameters in query
         List<Parameter> parameters = getParameters(statement);
 
-        // return the positions and types of all parameters
+        // return the positions and types of all parameters.  If there are no parameters, return a single null row.
         Row[] rows = parameters.stream().map(parameter -> createDescribeInputRow(parameter, queryAnalysis)).toArray(Row[]::new);
+        if (rows.length == 0) {
+            rows = new Row[] {row(new NullLiteral(), new NullLiteral())};
```

Comment:
eh.  null is a more standard way to indicate absence of value.  Especially since one of the columns is supposed to be an int column.  As long as the behavior is clearly defined, it should be easy enough for anybody to take a null value and convert to an empty string if they need it for something.